flink/v1.12/flink/src/test/java/com/netease/arctic/flink/catalog/TestCatalog.java

```diff
@@ -18,65 +18,70 @@
 
+import com.netease.arctic.TableTestHelper;
+import com.netease.arctic.ams.api.properties.TableFormat;
+import com.netease.arctic.catalog.BasicCatalogTestHelper;
+import com.netease.arctic.catalog.CatalogTestBase;
+import com.netease.arctic.flink.catalog.descriptors.ArcticCatalogValidator;
+import org.apache.flink.api.common.restartstrategy.RestartStrategies;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.runtime.state.StateBackend;
+import org.apache.flink.runtime.state.filesystem.FsStateBackend;
+import org.apache.flink.streaming.api.CheckpointingMode;
+import org.apache.flink.streaming.api.environment.CheckpointConfig;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.TableResult;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
 import org.apache.flink.types.Row;
+import org.apache.flink.util.CloseableIterator;
 import org.apache.flink.util.CollectionUtil;
-
-import com.netease.arctic.flink.FlinkTestBase;
-import com.netease.arctic.flink.table.ArcticTableLoader;
-import com.netease.arctic.table.ArcticTable;
-import com.netease.arctic.table.TableIdentifier;
-import org.apache.iceberg.UpdateProperties;
+import org.apache.iceberg.flink.MiniClusterResource;
+import org.apache.iceberg.relocated.com.google.common.collect.Lists;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
 import java.io.IOException;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.ExecutionException;
 
-import static com.netease.arctic.ams.api.MockArcticMetastoreServer.TEST_CATALOG_NAME;
-import static com.netease.arctic.flink.table.descriptors.ArcticValidator.LOG_STORE_CATCH_UP;
-import static com.netease.arctic.flink.table.descriptors.ArcticValidator.LOG_STORE_CATCH_UP_TIMESTAMP;
+import static org.apache.flink.table.api.config.TableConfigOptions.TABLE_DYNAMIC_TABLE_OPTIONS_ENABLED;
 
-public class TestCatalog extends FlinkTestBase {
+public class TestCatalog extends CatalogTestBase{
 
   @Rule
   public TemporaryFolder tempFolder = new TemporaryFolder();
+  protected Map<String, String> props;
 
-  private static final String DB = PK_TABLE_ID.getDatabase();
-  private static final String TABLE = "test_keyed";
+  private static final String DB = TableTestHelper.TEST_DB_NAME;
+  private static final String TABLE = TableTestHelper.TEST_TABLE_NAME;
 
-  public void before() throws Exception {
-    super.before();
-    super.config(TEST_CATALOG_NAME);
+  private volatile StreamExecutionEnvironment env = null;
+  private volatile StreamTableEnvironment tEnv = null;
+
+  public TestCatalog() {
+    super(new BasicCatalogTestHelper(TableFormat.MIXED_ICEBERG));
   }
 
-  @Test
-  public void testRefresh() {
```

Comment:
Why is this method discarded?

Reply:
I have moved the testRefresh() to a new TestClass named TestTableRefresh.